packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -2259,3 +2259,97 @@ export class Swap extends Op {
     stack.push(b);
   }
 }
+
+/**
+ * bit indexing begins with low-order bits in integers.
+ * Setting bit 4 to 1 on the integer 0 yields 16 (int 0x0010, or 2^4).
+ * Indexing begins in the first bytes of a byte-string
+ * (as seen in getbyte and substring). Setting bits 0 through 11 to 1
+ * in a 4 byte-array of 0s yields byte 0xfff00000
+ */
```

Comment:
please also add in description what this opcodes pops & pushes to stack

Reply:
ok